force-app/main/default/lwc/attendanceRow/attendanceRow.js

```diff
@@ -0,0 +1,94 @@
+import { LightningElement, api, track, wire } from "lwc";
+import { getRecord } from "lightning/uiRecordApi";
+import SERVICEDELIVERY_OBJECT from "@salesforce/schema/ServiceDelivery__c";
+import QUANTITY_FIELD from "@salesforce/schema/ServiceDelivery__c.Quantity__c";
+import CONTACT_NAME_FIELD from "@salesforce/schema/Contact.Name";
+import CONTACT_FIELD from "@salesforce/schema/ServiceDelivery__c.Contact__c";
+import ATTENDANCE_STATUS_FIELD from "@salesforce/schema/ServiceDelivery__c.AttendanceStatus__c";
+
+// TODO: create design parameters for default status and "present" statuses
+const PRESENT_STATUS = "Present";
+
+export default class AttendanceRow extends LightningElement {
+    @track localRecord;
+    @track localFieldSet;
+
+    @api unitOfMeasurement;
+    @api contactId;
+    @api presentStatus = PRESENT_STATUS;
+    oldStatus;
+    name;
+
+    @api
+    get record() {
+        return this.localRecord;
+    }
+    set record(value) {
+        this.localRecord = Object.assign({}, value);
+        this.contactId = this.localRecord[CONTACT_FIELD.fieldApiName];
+    }
+
+    @api
+    get fieldSet() {
+        return this.localFieldSet;
+    }
+    set fieldSet(value) {
+        this.localFieldSet = value;
+        if (value && value.length) {
+            this.setValues();
+        }
+    }
+
+    fields = {
+        quantity: QUANTITY_FIELD,
+        status: ATTENDANCE_STATUS_FIELD,
+    };
+
+    serviceDeliveryObject = SERVICEDELIVERY_OBJECT;
+
+    get recordId() {
+        return this.record.Id;
+    }
+
+    get quantity() {
+        return this.record[this.fields.quantity.fieldApiName];
+    }
+
+    @wire(getRecord, {
+        recordId: "$contactId",
```

Comment:
I am still wondering if we are better off getting everything in Apex in one trip and passing it down. Maybe a good topic for dev sync?